### PR TITLE
tds-protocol.md update

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
Updated the tds-protocol.md page; removed conversational content that didn't provide technical information, removed links to proprietary content and visuals, added links to neutral online sources, corrected voice/grammar

### Description
Updated the tds-protocol.md page to remove content that didn't provide technical information or problem resolution, removed links to proprietary content and visuals, added links to neutral (non-commercial) online resources about networking, corrected voice/grammar.  

My primary concern is that our sparse coverage of this huge topic was too open-ended, and didn't actually help the user resolve issues they might run into.  If I've removed important content that the user needs, please let me know!

### Issues Resolved
See above.

### Check List
- [x ] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by rcv-aws — DISMISSED** [review]

File: _internals/tds-protocol.md [hunk 1]
```diff
@@ -6,84 +6,40 @@ nav_order: 8
...
-As a database protocol, TDS relies on network transmission order and expects a
-reliable connection which basically means ...
-
-- TCP
+- [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol)
 - VIA (Virtual Interface Architecture)
-- Named pipes
+- [Named pipes](https://en.wikipedia.org/wiki/Named_pipe)
+- Transport Layer Security [(TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) with Secure Socket Layer [(SSL)](https://datatracker.ietf.org/doc/html/rfc6101)
+- Session Multiplex Protocol [(SMUX)](https://www.w3.org/TR/WD-mux), implemented with Multiple Active Result Sets (MARS)
```
Comment (susanmdouglas-aws) [inline _internals/tds-protocol.md:15]: Emailing with Rob re: SMUX/MARS...

File: _internals/tds-protocol.md [hunk 2]
```diff
@@ -6,84 +6,39 @@ nav_order: 8
...
-basic communication exchange on a high level.
-
-As a database protocol, TDS relies on network transmission order and expects a
-reliable connection which basically means ...
-
-- TCP
+- [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol)
 - VIA (Virtual Interface Architecture)
-- Named pipes
+- [Named pipes](https://en.wikipedia.org/wiki/Named_pipe)
```
Comment (rcv-aws) [inline _internals/tds-protocol.md:13]: Named pipes are NOT supported (I checked with Kuntal)

**Review by rcv-aws — CHANGES_REQUESTED** [review]
Remove named pipes -- Kuntal confirms we do not support those.

File: _internals/tds-protocol.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _internals/tds-protocol.md:13]: Removed... thank you!

**Review by kuntalghosh — APPROVED** [review]

**Comment by susanmdouglas-aws** [comment]
Removed named pipes, per Rob/Kuntal... thanks!

**Review by rcv-aws — APPROVED** [review]

**Review by 3manuek — APPROVED** [review]

**Pull request merged**